--- apertium-cy-en-0.1.1/configure.ac.orig	2013-05-31 22:40:52.000000000 +0200
+++ apertium-cy-en-0.1.1/configure.ac	2015-12-28 22:06:40.748836882 +0100
@@ -32,8 +32,13 @@
                   APERTIUM_VER=31
               ],
               [
-                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0)
+                  PKG_CHECK_MODULES(APERTIUM, apertium-3.2 >= 3.2.0,
+                  [
                   APERTIUM_VER=32
+		  ],
+		  [PKG_CHECK_MODULES(APERTIUM, apertium >= 3.3.0)
+		   APERTIUM_VER=3
+		  ])
               ])
           ])
       ])
